pyproject.toml

```diff
@@ -199,6 +199,7 @@ label-studio-converter = "0.0.57"
 google-cloud-storage = "^2.13.0"
 mysqlclient = {version = "*", optional = true}
 django-csp = "3.7"
+pyuwsgi = "~2.0.23"
```

Comment:
mind caret-pinning this instead for more flexibility?

Reply:
also perhaps we should move this into a separate `docker` dependency group since I don't think it's needed outside of docker; then in dockerfile we can add a `poetry install --only docker` somewhere.